source/models/tasks_model.py

```diff
@@ -32,9 +32,11 @@ def __init__(self, id_task_status=None, description=None):
 
 class TaskOwner(db.Model):
     __tablename__ = 'task_owners'
-    id_task_owner = db.Column(Integer, primary_key=True)
-    id_task = db.Column(Integer, ForeignKey('tasks.id_task'))
-    id_user = db.Column(Integer, ForeignKey('users.id_user'))
+    id_task_owner = db.Column(Integer, ForeignKey('users.id_user'), primary_key=True)
+    id_task = db.Column(Integer, ForeignKey('tasks.id_task'), primary_key=True)
+    id_user = db.Column(Integer, ForeignKey('users.id_user'), primary_key=True)
+    owner = db.relationship('User', foreign_keys=[id_task_owner], backref=db.backref('tasks owner per task'))
```

Comment:
Try to name backrefs as a sinhgle string, you can use that phrase but instead of spaces use underscores `tasks_owner_per_tasks` and same goes to the backref in line 39